packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/ReconnectionManager.kt

```diff
@@ -19,11 +19,11 @@ interface ReconnectionManagerListener {
   }
 }
 
+@Suppress("ktlint:standard:enum-entry-name-case")
 enum class ReconnectionStatus {
-  IDLE,
-  RECONNECTING,
-  ERROR,
-  WAITING
+  idle,
+  reconnecting,
+  error
 }
```

Comment:
WDYT about sth like:
```
enum class ReconnectionStatus(val status: String) {
  IDLE("idle"),
  RECONNECTING("reconnecting"),
  ERROR("error")
}
```

Reply:
All caps enum values are discouraged in Kotlin. I did this change so it looks better while being simpler and because PeerStatus was already done this way.

However it's true that first letters should be uppercased if I want to follow Kotlin rules.
I can change this to:
```
enum class ReconnectionStatus(val status: String) {
  Idle("idle"),
  Reconnecting("reconnecting"),
  Error("error")
}
```